systemtest/src/main/java/io/strimzi/systemtest/parallel/SuiteThreadController.java

```diff
@@ -252,7 +252,7 @@ public void decrementParallelSuiteCounter() {
      * @param extensionContext extension context for identifying, which test suite notifies.
      */
     public void notifyParallelSuiteToAllowExecution(ExtensionContext extensionContext) {
-        LOGGER.debug("{} - Notifies waiting test suites:{} to and randomly select one to start execution", extensionContext.getRequiredTestClass().getSimpleName(), waitingTestSuites.toString());
+        LOGGER.debug("{} - Notifies waiting test suites: {} to and randomly select one to start execution", extensionContext.getRequiredTestClass().getSimpleName(), waitingTestSuites.toString());
```

Comment:
You have `TestSuite` on one place, `test suites` here
I would make it consistent

Reply:
there are just a few instances of test suites, and all of them are logged like this. Would you like to make them all TestSuite ?